sonar-project.properties

```diff
@@ -1,8 +1,13 @@
 sonar.projectKey=snl-frontend
```

Comment:
Few things for this file:
- I would use just 'src' for both sources and tests to include everything
- This two fields will keep it in sync with naming strategy used for java projects
```
sonar.projectKey=snl-frontend
sonar.projectName=Reform :: Scheduling & Listing :: snl-frontend POC build extension
```
- `sonar.language=ts`